app/src/main/java/com/kickstarter/ui/activities/compose/search/SearchScreen.kt

```diff
@@ -248,12 +283,15 @@ fun SearchScreen(
                         },
                         selectedFilterCounts = selectedFilterCounts,
                         onSortPressed = {
-                            // Add logic here
+                            activeBottomSheet.value = "sort"
```

Comment:
Now `activeBottomSheet.value = FilterRowPillType.SORT` and similar change on line 292